components/NewFooter/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { FormattedMessage } from 'react-intl';
+import { FaGithub, FaSlack, FaTwitter, FaMastodon } from 'react-icons/fa';
+import styles from './index.module.scss';
+import LocalizedLink from '../LocalizedLink';
+
+const Footer = () => (
+  <footer className={styles.footer}>
+    <ul className={styles.left}>
+      <li>
+        <a
+          className={styles.link}
+          target="_blank"
+          href="https://trademark-policy.openjsf.org/"
+          rel="noopener noreferrer"
+          aria-label="Node.js Trademark Policy - PDF (Opens in new tab)"
+        >
+          <FormattedMessage id="components.footer.links.trademark" />
+        </a>
+      </li>
+      <li>
+        <a
+          className={styles.link}
+          target="_blank"
+          href="https://privacy-policy.openjsf.org/"
+          rel="noopener noreferrer"
+          aria-label="Node.js Privacy Policy - PDF (Opens in new tab)"
+        >
+          <FormattedMessage id="components.footer.links.privacy" />
+        </a>
+      </li>
+      <li>
+        <a
+          className={styles.link}
+          href="https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct"
```

Comment:
Shouldn't we add target as `_blank` and `rel="noopener noreferrer"` here 🤔

Reply:
yeah

Reply:
Done!